quipucords/scanner/satellite/six.py

```diff
@@ -332,6 +291,113 @@ def host_subscriptions(scan_task, url, org_id, host_id, ssl_cert_verify=None,
     return subs_dict
 
 
+def log_message(message, inspect_task_id, scan_type,
+                log_level=logging.INFO):
+    """Log a message for the information provided.
+
+    :param message: A string containing the message
+    :param inspect_task_id: The id of the running task
+    :param scan_type: The type of scan running
+        (inspect, connect)
+    :param log_level: The level to log at
+    """
+    # elapsed_time = self._compute_elapsed_time()
+    actual_message = 'Job %d (%s, elapsed_time:) - ' % \
+        (inspect_task_id,
+         scan_type)
+    actual_message += message
+    logger.log(log_level, actual_message)
+
+
+# pylint: disable=too-many-arguments
+def get_http_responses(scan_task, inspect_task_id,
+                       scan_type, host_id, host_name,
+                       fields_url, subs_url, request_options):
+    """Execute both http responses to gather satallite data.
+
+    :param scan_task: The current scan task
+    :param inspect_task_id: The id of the running task
+    :param scan_type: The type of scan running
+        (inspect, connect)
+    :param host_id: The id of the host we're inspecting
+    :param host_name: The name of the host we're inspecting
+    :param fields_url: The sat61 or sat62 fields url
+    :param subs_url: The sat61 or sat62 subs url
+    :param request_options: A dictionary containing host, port,
+        ssl_cert_verify, user, and password
+    :returns: A list containing the unique name for the host,
+        the response & url for host_fields request, and the
+        response & url for the host_subs request.
+    """
+    unique_name = '%s_%s' % (host_name, host_id)
+    host_fields_response = None
+    host_fields_url = None
+    host_subscriptions_response = None
+    host_subscriptions_url = None
+    try:
+        message = 'REQUESTING HOST DETAILS: %s' % unique_name
+        log_message(message, inspect_task_id, scan_type)
+        host_fields_response, host_fields_url = \
```

Comment:
I would move some of the basic request processing into this method.  For example, check status code.  Convert response body to a JSON dict and return that instead of the raw response.